soccer/src/soccer/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Make sure that the properties that you add on vscode are not pushed to the main branch!